python/pyspark/sql/connect/functions/__init__.py

```diff
@@ -16,7 +16,7 @@
 #
 
 """PySpark Functions with Spark Connect"""
-from pyspark.testing import should_test_connect
+from pyspark.testing.utils import should_test_connect
```

Comment:
Ditto. This change `python/pyspark/sql/connect/functions/__init__.py` which is a main source code.